game/main.cpp

```diff
@@ -0,0 +1,33 @@
+// This file is a plaground for testing out the game interface
+
+#include "interface.h"
+
+#include <cstdio>
+#include <cstdint>
+
+V<uint32_t, 0x50C044> debug_mode;
+V<uint32_t, 0xE98EB0> digital_input;
+struct foo {
+  uint32_t x;
+};
+S<foo, 0xF00BAA> s;
```

Comment:
Will be replaced with an actual pointer to something useful, using FOOBAR for testing